src/agent/alerting.py

```
@@ -0,0 +1,164 @@
+    """
+    db = get_db()
+
+    # Determine recipient
+    recipient = _get_stakeholder_email(campaign.target_market)
+
+    # 1. Send email (if configured)
+    if hasattr(settings, "SMTP_HOST") and settings.SMTP_HOST:
+        try:
+            await _send_email(
+                to=recipient,
+                subject=f"⚠️ Asset Generation Delay – {campaign.name or campaign.id}",
```

Ruff (0.13.3)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)